modules/odc-uninstalling-web-terminal.adoc

```diff
@@ -0,0 +1,74 @@
+
+.Prerequisites
+* Access to an OpenShift Container Platform cluster using an account with `cluster-admin` permissions.
+
+== Deleting the web terminal components and custom resources
+
+Use the CLI to delete the Custom Resources (CRs) created by default during installation of the  *Web Terminal* Operator.
+
+.Procedure
+. Ensure that all `DevWorkspace` CRs are removed along with their related Kubernetes objects, such as, deployments. It is crucial that this is done first, otherwise finalizers make it difficult to fully uninstall the operator.
++
+----
+oc delete devworkspaces.workspace.devfile.io --all-namespaces --all --wait
+oc delete workspaceroutings.controller.devfile.io --all-namespaces --all --wait
+oc delete components.controller.devfile.io --all-namespaces --all --wait
+----
++
+. Remove the custom resource definitions:
++
+----
+oc delete customresourcedefinitions.apiextensions.k8s.io workspaceroutings.controller.devfile.io
+oc delete customresourcedefinitions.apiextensions.k8s.io components.controller.devfile.io
+oc delete customresourcedefinitions.apiextensions.k8s.io devworkspaces.workspace.devfile.io
+----
++
+. Remove the `DevWorkspace-Webhook-Server` deployment:
++
+----
+oc delete deployment/devworkspace-webhook-server -n openshift-operators
+----
++
+[NOTE]
+====
+When you run this and the following steps, you cannot use the `oc exec` commands to run commands in a container. Once you remove the webhooks you will be able to use the `oc exec` commands again.
+====
++
+. Remove any lingering service, secrets, and configmaps:
++
+----
+oc delete all --selector app.kubernetes.io/part-of=devworkspace-operator,app.kubernetes.io/name=devworkspace-webhook-server
+oc delete serviceaccounts devworkspace-webhook-server -n openshift-operators
+oc delete configmap devworkspace-controller -n openshift-operators
+oc delete clusterrole devworkspace-webhook-server
+oc delete clusterrolebinding devworkspace-webhook-server
+----
++
+. Remove mutating or validating webhook configurations:
++
+----
+oc delete mutatingwebhookconfigurations controller.devfile.io
+oc delete validatingwebhookconfigurations controller.devfile.io
+----
+
+== Uninstalling the Operator using the web console
+
+.Procedure
+
+. In the *Administrator* perspective of the web console, navigate to *Operators → Installed Operators*,
+. Scroll or type a keyword into the *Filter by name* box to find the *Web Terminal* Operator.
+. Click the Options menu {kebab} on the right-hand side of the listed Operator, and select *Uninstall Operator* to see the *Uninstall Operator?* dialog box.
+. Select *Uninstall* to remove the Operator, Operator deployments, and pods from the cluster. The Operator stops running and no longer receives updates.
```

Comment:
I would actually recommend re-writing the last two steps here a bit. Along these lines:
````
. Click the Options menu {kebab} for the Web Terminal Operator, and then select *Uninstall Operator*.
. In the *Uninstall Operator* dialog box, click *Uninstall*.

The Operator, Operator deployments, and pods are removed from the cluster. The Operator stops running and no longer receives updates.
````